README.md

```diff
@@ -166,6 +166,8 @@ Options:
 include("<path>"[, <dynamicLocalVariables>...])
 ```
 
+[Java code](https://github.com/search?q=repo%3Ametafacture%2Fmetafacture-fix+path%3A%2F%5Emetafix%5C%2Fsrc%5C%2Fmain%5C%2Fjava%5C%2Forg%5C%2Fmetafacture%5C%2Fmetafix%5C%2FFixMethod.java%2F+include&type=code)
```

Comment:
1. Inconsistent capitalization (only here).
2. Exact matching (best to tighten everywhere).
3. Simpler path qualifier (file names are unique).

```suggestion
[Java Code](https://github.com/search?q=repo:metafacture/metafacture-fix+path:FixMethod.java+"+include+{"&type=code)
```

Reply:
+1 I will use this everywhere